docs/setup.md

```diff
@@ -54,6 +54,7 @@ Copy the `config.sample.yml` to a new file `config.yml`. The sample config is al
 [here](./setup/sample-configuration.md) for your convenience.
 
 You should read and fill this in as the bridge will not start without a complete config.
+Sections and items marked as `Optional` need to either be configured if you want to use the respective feature, or removed/commented out to disable the feature.
```

Comment:
The wording "need" is not technically 100% precisely correct, but at least I think it works and is not wrong. See discussion at #431.

This remark could additionally be added to the respective service's pages, if required for visibility

Reply:
"can" seems to fit here?